packages/form/src/TextField/TextField.js

```diff
@@ -76,7 +79,7 @@ class TextField extends Component<Props> {
         <div className="a-input__wrapper">
           {iconleft ? <Icon name={iconleft}/> : null}
           {mask ? <InputMask {...this.props} /> :
-            <input id={id} type="text" autoComplete={autoComplete} disabled={disabled}
+            <input id={id} type={type} autoComplete={autoComplete} disabled={disabled}
```

Comment:
omdat je hier "text" door type vervangt zou ik in de destructuring van de props type default op "text" zetten

Reply:
Had dezelfde opmerking